task-1-solution/FirstTaskSolution/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,359 @@
+            android:textSize="14sp">
+
+        </TextView>
+
+
+        <ImageView
+            android:id="@+id/nextSecondDayWeatherImage"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_centerInParent="true"
+            android:src="@drawable/ic_cloudy">
+
+        </ImageView>
+
+        <TextView
+            android:id="@+id/nextSecondDayCelciusText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:fontFamily="@font/roboto_medium"
+            android:layout_marginEnd="0dp"
+            android:text="14° - 27°"
+            android:textColor="@color/light"
+            android:textSize="14sp">
+
+        </TextView>
+
+
+    </RelativeLayout>
+
+
+    <RelativeLayout
+        android:id="@+id/nextThirdDayRelative"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:layout_marginLeft="24dp"
+        android:layout_marginRight="24dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/nextSecondDayRelative">
+
+
+        <TextView
+            android:id="@+id/nextThirdDayDateText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_centerVertical="true"
+            android:layout_marginEnd="0dp"
+            android:fontFamily="@font/roboto_regular"
+            android:layout_marginBottom="0dp"
+            android:text="Per, 04 Nisan"
+            android:textColor="@color/light"
+            android:textSize="14sp">
+
+        </TextView>
+
+
+        <ImageView
+            android:id="@+id/nextThirdDayWeatherImage"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_centerInParent="true"
+            android:src="@drawable/ic_sunny">
+
+        </ImageView>
+
+        <TextView
+            android:id="@+id/nextThirdDayCelciusText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:fontFamily="@font/roboto_medium"
+            android:layout_marginEnd="0dp"
+            android:text="14° - 27°"
+            android:textColor="@color/light"
+            android:textSize="14sp">
+
+        </TextView>
+
+
+    </RelativeLayout>
+
+
+    <RelativeLayout
+        android:id="@+id/nextFourthDayRelative"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:layout_marginLeft="24dp"

```

Comment:
left right kullanımları RTL diller için sorun olabiliyor, bunları start-end şeklinde kullanabiliriz

Reply:
tamamdır